app/gui/view/component-browser/component-group/src/wide.rs

```diff
@@ -0,0 +1,432 @@
+//! A multi-column [Component Group] without header.
+//!
+//! The widget is defined by the [`View`].
+//!
+//! To learn more about component groups, see the [Component Browser Design
+//! Document](https://github.com/enso-org/design/blob/e6cffec2dd6d16688164f04a4ef0d9dff998c3e7/epics/component-browser/design.md).
+//!
+//! [Component Group]: crate::component_group::View
+
+use ensogl_core::application::traits::*;
+use ensogl_core::display::shape::*;
+use ensogl_core::prelude::*;
+
+use crate::EntryId;
+
+use enso_frp as frp;
+use ensogl_core::application::shortcut::Shortcut;
+use ensogl_core::application::Application;
+use ensogl_core::data::color::Rgba;
+use ensogl_core::display;
+use ensogl_gui_component::component;
+use ensogl_label::Label;
+use ensogl_list_view as list_view;
+use list_view::entry::AnyModelProvider;
+
+
+// =================
+// === Constants ===
+// =================
+
+const NO_ITEMS_LABEL_TEXT: &str = "No local variables";
```

Comment:
1. It's not a sentence.
2. This is a component group item, a generic thing, what are "local variables"?

Reply:
(2) Is implemented as per design doc: 
> In case there are no local variables, this section should be displayed with a text "No local variables" inside.

I made this text configurable with an additional FRP input though.

Reply:
Ok, this should have dot in the end, that's a mistake in the design doc. Regarding point 2, ugh, I'm sorry, I was totally wrong. I was sure this is a generic component, not a component-browser only. I overlooked the name of the module. It was totally fine, I'm sorry for that.